### PR TITLE
Replaced delegation list displayed under validator page with total number of delegations

### LINKED ISSUE #484: [stargate] Delegations under validator page doesn't show all delegations
It's currently only showing 100 delegations without sorting under validator page. Delegations should be sorted in descending order.

### PR DESCRIPTION
**Pull request by MonikaCat**
## Description

Fixes #484

## Checklist
- [x] Targeted PR against correct branch.
- [x] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [x] Run `meteor npm run lint`
- [x] Added an entry to the `CHANGELOG.md` file.
- [x] Re-reviewed `Files changed` in the Github PR explorer.

### CODE REVIEW

**Comment by kwunyeung** [comment]
@MonikaCat nice one. Can it become a pagination component instead of showing a list of fixed 500 delegations?

File: imports/api/validators/server/methods.js [hunk 1]
```diff
@@ -26,7 +26,7 @@ Meteor.methods({
     // async 'Validators.getAllDelegations'(address){
     'Validators.getAllDelegations'(address){
         this.unblock();
-        let url = API + '/cosmos/staking/v1beta1/validators/'+address+'/delegations';
+        let url = API + '/cosmos/staking/v1beta1/validators/' + address +'/delegations?pagination.limit=500';
```
Comment (kwunyeung) [inline imports/api/validators/server/methods.js:29]: Make this query by page number.

**Review by kwunyeung — CHANGES_REQUESTED** [review]

File: imports/api/validators/server/methods.js (hunk 1 above)
Comment (MonikaCat) [reply imports/api/validators/server/methods.js]: I have added the changes that we talked about today in the office. As grpc is currently very slow when querying the list of delegations, I have replaced the delegation list displayed under the validator page with the total number of delegations for each validator.

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,7 +8,7 @@
 * Fixed an issue on displaying individual transaction.
 * Updated Ledger app name checking so that it will follows the value defined in settings.
 * [#213] Updated `@ledgerhq/hw-transport-webusb` pkg to v5.49.0 to fix Ledger errors on Windows10
-
+* [#484] Replaced delegation list displayed under validator page with total number of delegations.
```
Comment (kwunyeung) [inline CHANGELOG.md:11]: Please move to `[Unreleased]`.

**Review by kwunyeung — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (MonikaCat) [reply CHANGELOG.md]: Ok, done 👍🏻

**Review by kwunyeung — APPROVED** [review]

**Pull request merged**